index.html

```diff
@@ -7,21 +7,23 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <h1>This is page #1</h1>
-    <p>
-      Lorem ipsum, dolor sit amet consectetur adipisicing elit. Vel nam dolorem
-      ducimus ullam commodi. Officia, eum repellendus recusandae ullam pariatur
-      dicta amet fuga molestiae. Omnis nemo laboriosam fugit perspiciatis
-      distinctio.
-    </p>
-    <div class="pagination">
-      <a href="index.html"><</a>
-      <a href="index.html" class="active">1</a>
-      <a href="page2.html">2</a>
-      <a href="page3.html">3</a>
-      <a href="page4.html">4</a>
-      <a href="page5.html">5</a>
-      <a href="page2.html">></a>
+    <div class="gallery">
+      <a target="_blank" href="images/food 1.jpg">
+        <img src="images/food 1.jpg" alt="salad" />
```

Comment:
Don't use spaces in file names 

`<img src="images/food_1.jpg" alt="salad" />`